build.gradle

```diff
@@ -74,43 +67,142 @@ subprojects {
-        into "$ghpages/docs/${rootProject.version}"
+    task sourcesJar(type: Jar, dependsOn: classes) {
+        classifier = 'sources'
+        sourceSets.all {
+            into(name + "/java", { from allJava })
+            into(name + "/resources", { from resources })
+        }
     }
 
-    task publishDocsLatest(type: Sync, dependsOn: docs) {
-        from "build/docs"
-        into "$ghpages/docs/latest"
+    artifacts {
+        archives sourcesJar, javadocJar
     }
 
-    // Publish docs and maven artifacts to GH repo
-    task publishAll(dependsOn: [publishDocsVersioned, publishDocsLatest, subprojects.publish])
+    uploadArchives {
+        ext.ossrhUsername = System.properties.ossrhUsername
+        ext.ossrhPassword = System.properties.ossrhPassword
+
+        doFirst {
+            // If the OSSRH credentials are not available, then fail the build
+            if (ossrhUsername == null || ossrhPassword == null) {
+                throw new GradleException('OSSRH credentials properties (ossrhUsername & ' +
+                        'ossrhPassword) are required to upload archives.')
+            }
+        }
+
+        repositories {
+            mavenDeployer {
+                //when publishing sign the pom
+                beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+
+                repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
+                    authentication(userName: ossrhUsername, password: ossrhPassword)
+                }
+
+                snapshotRepository(url: "https://oss.sonatype.org/content/repositories/snapshots/") {
+                    authentication(userName: ossrhUsername, password: ossrhPassword)
+                }
+
+                //augment the pom with additional information
+                pom.project {
+                    packaging 'jar'
+                    inceptionYear '2013'
+                    url 'https://cloudant.com'
+                    licenses {
+                        license {
+                            name 'The Apache Software License, Version 2.0'
+                            url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                            distribution 'repo'
+                        }
+                    }
+                    scm {
+                        connection 'scm:git:git://github.com/cloudant/sync-android.git'
+                        developerConnection 'scm:git:git@github.com/cloudant/sync-android.git'
+                        url 'https://java-cloudant@github.com/cloudant/sync-android.git'
+                    }
+                    properties {
+                        'project.build.sourceEncoding' 'UTF-8'
+                    }
+                    developers {
+                        developer {
+                            name 'IBM Cloudant'
+                            email 'support@cloudant.com'
+                            url 'https://cloudant.com'
```

Comment:
I didn't think we could get more specific than cloudant.com for the developer name